app/dockerdwrapper.c

```diff
@@ -694,35 +709,15 @@ main(void)
   ax_parameter = setup_axparameter();
   if (ax_parameter == NULL) {
     syslog(LOG_ERR, "Error in setup_axparameter: %s", error->message);
+    exit_code = -1;
     goto end;
   }
 
   /* Create the GLib event loop. */
   loop = g_main_loop_new(NULL, FALSE);
   loop = g_main_loop_ref(loop);
 
-  bool use_sdcard = false;
-  bool use_tls = false;
-  bool use_ipc_socket = false;
-
-  if (!get_and_verify_sd_card_selection(&use_sdcard)) {
-    syslog(LOG_INFO, "Failed to setup sd_card");
-    exit_code = -1;
-    goto end;
-  }
-  if (!get_and_verify_tls_selection(&use_tls)) {
-    syslog(LOG_INFO, "Failed to verify tls selection");
-    exit_code = -1;
-    goto end;
-  }
-  if (!get_ipc_socket_selection(&use_ipc_socket)) {
-    syslog(LOG_INFO, "Failed to get ipc socket selection");
-    exit_code = -1;
-    goto end;
-  }
-
-  if (!start_dockerd(use_sdcard, use_tls, use_ipc_socket)) {
-    syslog(LOG_ERR, "Starting dockerd failed");
```

Comment:
this error message got lost?

Reply:
start_dockerd already reports "Could not execv the dockerd process. Return value: %d, error: %s", but there is also another case with a comment "The process died during adding of callback, tell loop to quit.". I guess I could put this message there.